.flake8

```
@@ -0,0 +1,12 @@
+[flake8]
+
+# E501 line too long (93 > 88 characters)
+# F821 undefined name '_'
+
+exclude = .git
+
+max-line-length = 88
+
+per-file-ignores =
+    ./g.region.area.py: F821, E501
+    ./testsuite/test_g_region_area.py: F821
```

```suggestion
    ./testsuite/test_g_region_area.py: F821
```

(missing newline added)